packages/angular_devkit/build_angular/src/browser/index2.ts

```diff
@@ -0,0 +1,249 @@
+/**
```

Comment:
This builder really should be thoroughly tested as it's the most important one. 

Since the exposed functions are meant to be used by third party builders and thus constitute public API, they also need to be tested.

Reply:
Yes I should move all the tests for the browser builder. I'll do that asap.

Reply:
Although this is going well, there is a LOT of tests for this builder so it's taking me more time. The good news is that this is mostly manual work and there was only 1 bug found in architect so far. Should be done really soon.